README.md

```diff
@@ -46,4 +46,68 @@ const STARGATE_URL = "https://stargate.cosmos.network"
 const api = API(STARGATE_URL)
 
 const validators = await api.validators()
+```
+
+### Create a sign message to sign with on a Ledger or with any other signer
```

Comment:
```suggestion
### Create a message to be signed by a Ledger or with any other signer app
```

Reply:
does this make sense / is my suggestion true? it was hard to understand precisely what you meant.